modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -54,8 +61,8 @@ endif::[]
 ifeval::["{context}" == "installing-vsphere"]
 :vsphere:
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-vsphere"]
-:vsphere:
+ifeval::["{context}" == "installing-vmc-user-infra"]
+:vmc:
```

Comment:
So I think I see you defining ":vmc:" but not actually using it in this file

Reply:
I forgot to add it to one of the vsphere conditionals. Just added :+1: